app/controllers/batches_controller.rb

```diff
@@ -216,10 +216,6 @@ def save
 
   def print_labels; end
 
-  def print_stock_labels
-    @batch = Batch.find(params[:id])
-  end
-
```

Comment:
I think we can actually lose even more label printing stuff, but kept with the ones I'm more sure of. The rest of the ones still have links, they are just associated with pipelines where they make little sense. (And probably actually blow up)